### ISSUE TITLE
Merge preview branch into master

### ISSUE THREAD

**Issue by pablotolosa-aircury** [elemwave/website#23]
After the research in #13, we'll have to work with only one environment.

To differentiate between the production (master) and the preview for the client to accept will maintain the preview branch. Being preview the latest stable development and master the latest released version.

Both branches will have to be used in the only page we have, having the preview only for testing purposes:

1. If the changes are accepted it'll be merged into master, thus being both branches at the same point.
2. If not then the master branch can be deployed again to restore the previous accepted release.

To do in this card:

1. Create a PR to add preview into master, both branches will have to be the same. A review should be done before merging.
2. Update the documentation created in #16 to add this protocol to the workflow.

**Comment by pablotolosa-aircury** [comment]
.
